src/configs/os.rs

```diff
@@ -0,0 +1,77 @@
+use indexmap::{indexmap, IndexMap};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+#[cfg_attr(feature = "config-schema", derive(schemars::JsonSchema))]
```

Comment:
Please rebase/merge master and add `schemars(deny_unknown_keys)` here, like in the other config modules.